cli/pcluster/resources/batch/custom_resources_code/cleanup_resources.py

```diff
@@ -31,8 +31,7 @@ def delete_s3_bucket(bucket_name):
     Empty and delete the bucket passed as argument.
 
     It exits gracefully if bucket doesn't exist.
-    Args:
-        bucket_name: bucket to delete
+    :param bucket_name: bucket to delete
```

Comment:
What does getting rid of `Args` change?

Reply:
1. to uniform the docstring syntax 
2. the linter was complaining :)